apps/react-app/src/pods/template-export/template-export.styles.ts

```diff
@@ -48,7 +48,38 @@ export const textarea = css`
   color: ${theme.palette.light[700]};
 `;
 
+export const buttonContainer = css `
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: ${theme.spacing(4)};
+  width: 100%;
+
+  @media (max-width: 1024px) {
```

Comment:
Both media queries are the same; we don't need to repeat properties if the properties don't change. For example, display: flex is repeated three times. Update these styles, please.